lib/cocoapods/installer/analyzer/pod_variant.rb

```diff
@@ -30,12 +34,14 @@ def root_spec
         #
         # @param [Array<Specification>] specs      @see #specs
         # @param [Array<Specification>] test_specs @see #test_specs
+        # @param [Array<Specification>] app_specs  @see #app_specs
         # @param [Platform] platform               @see #platform
         # @param [Bool] requires_frameworks        @see #requires_frameworks?
         #
-        def initialize(specs, test_specs, platform, requires_frameworks = false)
+        def initialize(specs, test_specs, app_specs, platform, requires_frameworks = false)
```

Comment:
can you please update doc in line 50 about test specs + app specs not being considered for equality `==`?